internal/slicer/slicer.go

```diff
@@ -69,13 +71,48 @@ func Run(options *RunOptions) error {
 		targetDirAbs = filepath.Join(dir, targetDir)
 	}
 
+	// Archives are ordered in descending order by priority. The
+	// default priority is 0 and the maximum allowed priority is
+	// the upper limit of int.
```

Comment:
32-bit ints or 64-bit ints?
Additionally, could the maximum be some pretty value (10^x for example) instead of 2147483647 or 9223372036854775807?

Reply:
I've updated the comment with the change to int32. The range is now the same as the range of int32. This allows to lower archive priority from default 0. As for pretty decimal limits, I don't see the point of imposing artificial limits. :shrug: